Mitarbeiter.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from Vorgesetzter import Vorgesetzter
+
+
+class Mitarbeiter:
+
+    _bestelllimit: int = 20
+    _vorgesetzter: Vorgesetzter = None
+
+    def __init__(self, name) -> None:
+        self._name: str = name
+
+    @classmethod
+    def setzeAllgemeinesLimit(cls, limit: int) -> int:
+        cls._bestelllimit: int = limit
+        return cls._bestelllimit
+
+    def setzeVorgesetzten(self, name: Vorgesetzter) -> Vorgesetzter:
```

Comment:
```suggestion
    def setzeVorgesetzten(self, vorgesetzter: Vorgesetzter) -> Vorgesetzter:
```

Misleading parameter name. `name` suggests a string :)